scooter_fsm/scooter_fsm/states.py

```diff
@@ -13,10 +14,13 @@ def run(node, result):
         :return: Tuple containing the next state :class:`scooter_fsm.state.State` and the result of this state
         :rtype: tuple
         """
+        goal_handle = node.go_to_joint_config(JointConfigs.TRAVEL)
 
         request = WaitForBegin.Request()
         result = node.send_request(request)
 
+        success = node.get_joint_config_result(goal_handle)
```

Comment:
Do we not check for success or not here? I'm assuming this is actually a bool being returned but may be wrong. Error handling here?

Reply:
Yes, the action server should be returning a bool. I didn't add any sort of error handling here but we should probably discuss how we want to deal with errors in the FSM.